packages/core/src/node/backend-application.ts

```diff
@@ -37,6 +38,13 @@ export class BackendApplication {
                 }
             }
         });
+
+        for (const contribution of this.contributionsProvider.getContributions()) {
+            if (contribution.initialize) {
+                contribution.initialize();
```

Comment:
could be done separately, but we should also wrap calls in try-catch in the backend

Reply:
Right I'll make a PR for this now

Reply:
I'm adding this to this PR after all.. since it's quite trivial